src/xmpp/vcard.go

```diff
@@ -0,0 +1,15 @@
+package xmpp
+
+import (
+	"encoding/xml"
+)
+
+const (
+	NsVCardTemp = "vcard-temp"
+)
+
+// XEP-0054 vCard
+type VCard struct {
+	XMLName xml.Name `xml:"vcard-temp vCard"`
+	// TODO Must complete truct
```

Comment:
This doesn't look very useful at the moment. Maybe drop it from the branch for now?